pkg/iss/iss.go

```diff
@@ -309,7 +309,7 @@ func (iss *ISS) applySBInstDeliver(deliver *isspb.SBDeliver) (*events.EventList,
 	// might mismatch, if they commit in different PBFT views (and thus using different Preprepares).
 	logEntry := &CommitLogEntry{
 		Sn:       t.SeqNr(deliver.Sn),
-		CertData: deliver.CertData,
+		CertData: deliver.Data,
```

Comment:
The reason is that it's not just about certificates, but generic data (checkpoints) needs to be agreed upon.
Technically, what we really need to agree upon, happens to be a certificate (just one for the checkpoint), but
1. We still pass the whole serialized checkpoint to the agreement for the moment, just because it is simpler to implement and
2. Principally, the PBFT instance doesn't look at what it orders. From its perspective, it really only orders generic data, so this is more accurate.